NEWS.md

```diff
@@ -31,6 +31,8 @@
 
 * Fixed that `spread()` fails when the `key` column includes `NA` and `drop` is `FALSE` (#254).
 
+* `separate()` now works as described in documentation when using negative integer values with `sep` argument (#315).
```

Comment:
Can you please credit yourself with `(@yourname, #issuenumber)`?  And break the line to be <80 characters?